ynr/apps/elections/views.py

```diff
@@ -251,13 +251,31 @@ def form_valid(self, form):
                 # If we're locking this, then the suggested posts
                 # can be deleted
                 ballot.suggestedpostlock_set.all().delete()
+                if not ballot.uncontested:
+                    return None
```

Comment:
Don't want to return here as we still want lines 276 down to run

Reply:
Oops, should be `continue`